WalletConnectSharp.Core/Network/DefaultBridge.cs

```diff
@@ -59,7 +59,14 @@ public static string ChooseRandomBridge(string[] possibleBridges = null)
     public static string GetBridgeUrl(string url)
     {
         if (ExtractRootDomain(url) == Domain)
-            return ChooseRandomBridge();
+        {
+            var chosen = ChooseRandomBridge();
+            if (url.StartsWith("wss"))
+                chosen = chosen.Replace("https", "wss");
+            else if (url.StartsWith("ws"))
+                chosen = chosen.Replace("http", "ws");
```

Comment:
Just to get rid of the magic strings...

```suggestion
            if (url.StartsWith(Uri.UriSchemeWss))
                chosen = chosen.Replace(Uri.UriSchemeHttps, Uri.UriSchemeWss);
            else if (url.StartsWith(Uri.UriSchemeWs))
                chosen = chosen.Replace(Uri.UriSchemeHttp, Uri.UriSchemeWs);
```

Reply:
I like the idea of using the `Uri.UriSchemeWss` variables, however I think the variables may need to be imported or maybe their called something else. The build broke :(

Reply:
Yeah, it looks like [UriSchemeWSS](https://docs.microsoft.com/en-us/dotnet/api/system.uri.urischemewss) is .NET 6 forward, so netstandard2.1 isn't getting it just yet 😢 

Will need to define a static readonly class or a file-level constant to remove strings here.